ion-schema/src/test/kotlin/com/amazon/ionschema/internal/util/PreconditionsTest.kt

```diff
@@ -156,4 +162,226 @@ class PreconditionsTest {
             val ionInt: IonInt = maybeIonInt
         }
     }
+
+    @Nested
+    inner class islRequireElementType {
+        @Test
+        fun `when collection is empty, should return empty list`() {
+            val theList = emptyList<IonValue>()
+            val result = theList.islRequireElementType<IonValue>("a list")
+            assertEquals(theList, result)
```

Comment:
Looks like this could be replaced by a call to `expectOk` for consistency, if desired.